src/BCrypt/BCrypt+HashOperationType.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="HashOperationType"/> nested type.
+    /// </content>
+    public static partial class BCrypt
+    {
+        public enum HashOperationType
```

Comment:
We should use the [documented](https://docs.microsoft.com/en-us/windows/desktop/api/bcrypt/ns-bcrypt-_bcrypt_multi_hash_operation) name for this enum.

```suggestion
        public enum BCRYPT_HASH_OPERATION_TYPE
```

Reply:
I've taken care of this.